Improve Ballerina-file API doc (#22606)
ballerina-platform/ballerina-lang: stdlib/file/src/main/ballerina/src/file/Module.md

This module contains functions to perform file system based operations such as create, delete, rename the

> file/directory and retrieve metadata of the file.

Reviewer: ```suggestion
file/directory, and retrieve metadata of the file.
```

> For an example on the usage of file operations, see the [File Example](https://ballerina.io/v1-2/learn/by-example/file.html).

Reviewer: ```suggestion
For an example on the usage of the operations, see the [File Example](https://ballerina.io/learn/by-example/file.html).
```

## Directory Listener 

The `Directory Listener` is used to listen to a directory in the local file system. It notifies when new files are created in the directory or when the existing files are deleted or modified.

> For an example on the usage of `Directory Listener` see the [Directory Listener Example](https://ballerina.io/v1-2/learn/by-example/directory-listener.html).

Reviewer: ```suggestion
For an example on the usage of the operations, see the [Directory Listener Example](https://ballerina.io/learn/by-example/directory-listener.html).
```
